### PR TITLE
Sign main tag and add missing env during release workflow

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: jorturfer <jorge_turrado@hotmail.es>

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [x] Changelog has been updated

Related #2501

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

File: .github/workflows/release-build.yml [hunk 1]
```diff
@@ -77,6 +77,8 @@ jobs:
         # This step uses the identity token to provision an ephemeral certificate
         # against the sigstore community Fulcio instance.
         run: make sign-images
+        env:
+          VERSION: ${{ steps.get_version.outputs.VERSION }}
```
Comment (zroubalik) [inline .github/workflows/release-build.yml:81]: +1

**Review by zroubalik — COMMENTED** [review]
There is extra whitespace

**Comment by JorTurFer** [comment]
I'm AFK, I'll update it in 40-50 min.
I can merge it once it's green if you agree @zroubalik (or wait, we don't have rush at all)

**Comment by JorTurFer** [comment]
These f******* whitespaces... 🤣 🤣 🤣 
Solved

**Comment by zroubalik** [comment]
> These f******* whitespaces... 🤣 🤣 🤣 Solved

@JorTurFer btw you can enable pre-commit locally, so you catch these kind of issues before commiting the actual code.

**Comment by JorTurFer** [comment]
I got it... but disabled because the tools are only installed inside the dev container, I should install it also in the host. Probably when my new computer arrives, I'll do something more reliable xD (usually I run it manually, but sometimes I forget it)

Anyway, thanks for the suggestion ❤️

**Review by zroubalik — APPROVED** [review]

**Pull request merged**